src/test/java/Homework19.java

```diff
@@ -0,0 +1,27 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework19 extends BaseTest {
+
+@Test
+public void deletePlaylist() throws InterruptedException {
+login("guyvanalex4@gmail.com", "teststudent!1");
+playlist();
+verifyPlaylistDeleted();
+Thread.sleep(2000);
+}
+
+public void playlist() throws InterruptedException {
+    WebElement myPlaylist = driver.findElement(By.cssSelector("#playlists > ul > li:nth-child(3) > a"));
```

Comment:
1 - Please do not use plugins or copy XPath from page code using browser or web developer tools.
2 - Try to specify the locator of an exact element you want to find (for example you can find it by inner text)